.gitignore

```diff
@@ -31,6 +31,7 @@ master.key
 .env
 
 public/uploads
+public/contacts
```

Comment:
@sojan-official Need your thoughts on where we should save the file for contacts export.

Reply:
wouldn't need this folder / the file doesn't need to be stores in an open folder.